client/src/components/PinnedContent/PinnedContent.tsx

```diff
@@ -11,32 +11,54 @@ import { is_a11y_mode } from "src/core/injected_build_constants";
 import { IconPin, IconUnpin } from "src/icons/icons";
 import { backgroundColor } from "src/style_constants/index";
 
+export { NonA11yPinnedContent };
+
 import text from "./PinnedContent.yaml";
 
 const text_maker = create_text_maker(text);
 
-export const get_pinned_content_local_storage = (local_storage_name) => {
+interface NonA11yPinnedContentProps {
+  local_storage_name: string;
+  default_pin_state: boolean;
+}
+
+//Needed for PinnedFAQ to function correctly.
+interface PinnedContentProps {
+  local_storage_name: string;
+}
+
+interface isPinnedMirrorHelper {
```

Comment:
Nitpicking but have the name of the interface match the format of what we use for our other state interface names in other files

Reply:
Makes sense, will keep in mind for the future.